_sass/blocks/_landing.scss

```diff
@@ -68,40 +68,76 @@
   }
 }
 
-.landing-section + .h3-bar {
-  padding-top: 4rem;
-}
+.landing-section.landing-section-with-graphics {
+  .landing-heading {
+    position: relative;
+  }
 
-.landing-oil_gas,
-.landing-coal,
-.landing-minerals,
-.landing-renewables {
   svg {
-    float: right;
-    margin-right: 20px;
+    bottom: 0;
+    margin-right: 15px;
+    max-height: 100%;
+    position: absolute;
+    right: 0;
+    width: 55px;
 
     path {
       fill: $neutral-gray;
     }
   }
 
+  .landing-minerals {
+    svg {
+      margin-top: 46px;
+      top: -36px;
+    }
+  }
+
+  .landing-coal {
+    svg {
+      margin-right: 15px;
+      margin-top: 30px;
+      top: -20px;
+    }
+  }
+
   p {
-    float: left;
-    width: 57%;
+    width: auto;
   }
-}
+  @include respond-to(medium-up) {
```

Comment:
I disagree here houndci-bot as the approac is mobile first with the code at 72 is mobile first and the code at line 107 is for medium screens and up.